pkg/controller/machineautoscaler/machineautoscaler_controller.go

```diff
@@ -47,8 +47,10 @@ var (
 // SupportedTargetGVKs is the list of GroupVersionKinds supported as targets for
 // a MachineAutocaler instance.
 var SupportedTargetGVKs = []schema.GroupVersionKind{
-	{Group: "cluster.k8s.io", Version: "v1alpha1", Kind: "MachineSet"},
 	{Group: "cluster.k8s.io", Version: "v1alpha1", Kind: "MachineDeployment"},
+	{Group: "cluster.k8s.io", Version: "v1alpha1", Kind: "MachineSet"},
+	{Group: "machine.openshift.io", Version: "v1beta1", Kind: "MachineDeployment"},
```

Comment:
Not a blocker, this would belong to a different PR

Reply:
We need to decide quickly. this will merge RSN if the tests pass as we have LGTM and APPROVE.

Reply:
What belongs to a different PR? Just adding `MachineDeployments`? Or am I misunderstanding?

There's no harm in adding the deployments here.